core/src/main/java/io/questdb/network/IOContextFactoryImpl.java

```diff
@@ -63,7 +61,6 @@ public void freeThreadLocal() {
         Misc.free(this.contextPool);
     }
 
-    @Override
```

Comment:
annotations should not have been removed

Reply:
restored back

Reply:
thanks 👍